lib/rentlinx/client.rb

```diff
@@ -96,8 +96,7 @@ def response_handler(response)
       when 204
         nil # don't attempt to JSON parse emptystring
       when 400
-        body = JSON.parse(response.body)
-        raise Rentlinx::BadRequest, response, body['details']
+        raise Rentlinx::BadRequest, response
```

Comment:
yes, the 3rd param is an array of callback information, which doesn't seem to fit here